src/FurCoNZ.Web/Startup.cs

```diff
@@ -29,14 +29,11 @@
 using SendGrid;
 
 // Local
-using FurCoNZ.Web.Auth;
-using FurCoNZ.Web.Configuration;
-using FurCoNZ.Web.DAL;
-using FurCoNZ.Web.Services;
-
-namespace FurCoNZ.Web
-{
-    public class Startup
+using FurCoNZ.Web.Auth;
using FurCoNZ.Web.Configuration;
using FurCoNZ.Web.DAL;
+using FurCoNZ.Web.Options;
using FurCoNZ.Web.Services;
+
+namespace FurCoNZ.Web
+{
    public class Startup

```

Comment:
Did macOS use `\r` instead of `\n` or `\r\n` here?

Reply:
Really tired of macOS mangling line endings here. Not sure if it's Git or Visual Studio at fault here. Added a `.editorconfig` and `.gitattributes` to try and combat this, along with normalising line endings on `Settings.cs`.